cli/lib/kontena/cli/grids/remove_command.rb

```diff
@@ -5,24 +5,26 @@ class RemoveCommand < Kontena::Command
     include Kontena::Cli::Common
     include Common
 
-    parameter "NAME", "Grid name"
+    parameter "NAME ...", "Grid name", attribute_name: :grids
     option "--force", :flag, "Force remove", default: false, attribute_name: :forced
 
     def execute
       require_api_url
       token = require_token
-      confirm_command(name) unless forced?
-      grid = find_grid_by_name(name)
+      grids.each do |name|
+        confirm_command(name) unless forced?
+        grid = find_grid_by_name(name)
 
-      if !grid.nil?
-        spinner "removing #{pastel.cyan(name)} grid " do
-          response = client(token).delete("grids/#{grid['id']}")
-          if response
-            clear_current_grid if grid['id'] == current_grid
+        if !grid.nil?
+          spinner "removing #{pastel.cyan(name)} grid " do
+            response = client(token).delete("grids/#{grid['id']}")
+            if response
+              clear_current_grid if grid['id'] == current_grid
+            end
           end
+        else
+          exit_with_error "Could not resolve grid by name [#{name}]. For a list of existing grids please run: kontena grid list"
```

Comment:
I think these should just warn and continue instead of exit with error.

Consider something like:
```
$ kontena stack rm --force stack1 stack2 snack3 stack4
Could not resolve grid by name [snack3]  # <-- btw inconsistent formatting there with the []
```

After such an error you kind of have to guess / figure out which ones already went through.

Reply:
Hmm, but then again, how about: `kontena stack rm --force $(stack ls -q) && ...` maybe it's actually better to exit with error if even one fails.